addons/point_of_sale/models/pos_config.py

```diff
@@ -751,3 +665,14 @@ def _get_modules_to_check(self, changed_fields):
             if field.startswith('module_'):
                 modules_to_check.append(field.replace('module_', ''))
         return modules_to_check
+
+    def action_pos_config_modal_edit(self):
+        return {
+            'view_mode': 'form',
+            'view_id': self.env.ref('point_of_sale.pos_config_view_form').id,
```

Comment:
do we need to specify the view_id? I don't think there are multiple form views for pos.config